modules/mobile/spec/request/appointments_cancel_request_spec.rb

```diff
@@ -21,210 +21,268 @@
   after(:all) { VCR.configure { |c| c.cassette_library_dir = @original_cassette_dir } }
 
   describe 'PUT /mobile/v0/appointments/cancel' do
-    context 'confirmed appointments' do
-      context 'when request body params are missing' do
-        let(:cancel_id) do
-          'abc123'
-        end
-
-        it 'returns a 422 that lists all validation errors' do
-          VCR.use_cassette('appointments/get_cancel_reasons', match_requests_on: %i[method uri]) do
-            put "/mobile/v0/appointments/cancel/#{cancel_id}", headers: iam_headers
-
-            expect(response).to have_http_status(:unprocessable_entity)
-            expect(response.body).to match_json_schema('errors')
-            expect(response.parsed_body['errors'].size).to eq(1)
-          end
-        end
+    context 'when using VAOS v0 services' do
+      before do
+        Flipper.disable('mobile_appointment_use_VAOS_v2')
       end
 
-      context 'with valid params' do
-        let(:cancel_id) do
-          Mobile::V0::Appointment.encode_cancel_id(
-            start_date_local: DateTime.parse('2019-11-15T13:00:00'),
-            clinic_id: '437',
-            facility_id: '983',
-            healthcare_service: 'CHY VISUAL FIELD'
-          )
-        end
+      context 'confirmed appointments' do
+        context 'when request body params are missing' do
```

Comment:
Is this description accurate? It seems like the difference is that this is not a valid id.